src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -55,7 +55,7 @@ const SubmissionDetails = _.flow(
   useEffect(() => {
     const initialize = withErrorReporting('Unable to fetch submission details',
       async () => {
-        if (_.isEmpty(submission) || _.some(({ status }) => _.includes(collapseStatus(status), ['running', 'submitted']), submission.workflows)) {
+        if (_.isEmpty(submission) || _.some(({ status }) => _.includes(collapseStatus(status), [statusType.running, statusType.submitted]), submission.workflows)) {
           if (!_.isEmpty(submission)) {
             await Utils.delay(60000)
```

Comment:
@zarsky-broad it looks like you originally wrote this code. Do you recall what this delay is meant to do?

Reply:
If there are any in-progress submissions, it'll refresh the list every minute.

Reply:
Discussed in mob, and now I understand this much better. Thank you!